src/app/components/Sorter/Sorter.jsx

```diff
@@ -61,31 +75,54 @@ class Sorter extends React.Component {
     this.setState({ active: false });
   }
 
-  handleOnClickOut() {
-    if (this.state.active) {
-      this.setState({ active: false });
-    }
+  /**
+   * renderResultsSort()
+   * The fuction that makes renders the sort options.
+   *
+   * @return {HTML Element}
+   */
+  renderResultsSort() {
+    return sortingOpts.map((d, i) => (
+      <option value={d.val} key={i}>
+        {d.label}
+      </option>
+    ));
   }
 
   render() {
+    const keywords = this.props.keywords || '';
+    const field = this.props.field || '';
+
     return (
       <div className="nypl-results-sorting-controls">
         <div className="nypl-results-sorter">
-          <ClickOutHandler onClickOut={() => this.handleOnClickOut()}>
-            <button
-              aria-expanded={this.state.active}
-              className={this.state.active ? 'active' : ''}
-              onClick={e => this.getResultsWindow(e)}
-            >
-              <span>Sort by <strong>{this.state.sortLabel}</strong></span>
-              <DownWedgeIcon className="nypl-icon" viewBox="0 0 68 24" />
-            </button>
-            <ul className={this.state.active ? '' : 'hidden'}>
-              {
-                this.getResultsSort()
-              }
-            </ul>
-          </ClickOutHandler>
+          <form
+            action={
+              `/search${keywords ? `?q=${keywords}` : ''}${field ? `&search_scope=${field}` : ''}`
+            }
+            method="POST"
+          >
+            <span className="nypl-omni-fields">
+              <label htmlFor="search-by-field">Sort by</label>
+              <strong>
+                <select
+                  id="sort-by-label"
+                  onChange={this.updateSortValue}
+                  value={this.state.sortLabel}
+                  name="sort_scope"
```

Comment:
To continue with the other comments, the `value` for this gets picked up as `sort_scope` in the `req.body` property in `searchServerPost` function.